.gitignore

```diff
@@ -2,3 +2,4 @@ composer.lock
 vendor/
 node_modules/
 docs/
+public/
```

Comment:
Agree, fixed in commit: https://github.com/DivideBV/PHPDivideIQ/pull/1/commits/da56ac26dd7f0b358bf61df9266136ec8cd64516